src/test/java/org/folio/inventory/eventhandlers/MatchAuthorityEventHandlerUnitTest.java

```diff
@@ -458,6 +466,93 @@ public void shouldMatchWithSubMatchByAuthorityOnHandleEventPayload(TestContext t
     });
   }
 
+  @Test
+  public void shouldMatchWithSubConditionBasedOnMultiMatchResultOnHandleEventPayload(TestContext testContext) throws UnsupportedEncodingException {
+    Async async = testContext.async();
+    List<String> multiMatchResult = List.of(
+      "bd39d7cc-f313-47ea-bf2d-abcac2b24a64", "e6dc8015-fcad-40cf-afa9-e817a605dc06");
```

Comment:
Does the values of this list have to be the UUID listed? Could it be any UUID?

Reply:
they can be any UUID. Updated test by random UUID to avoid confusion regarding the purpose of these values.